security/MacPass/Portfile

```diff
@@ -72,6 +74,29 @@ xcode.destroot.settings CODE_SIGN_IDENTITY=- \
 destroot.pre_args-append  -derivedDataPath ./DerivedData
 destroot.args-append GCC_PREPROCESSOR_DEFINITIONS="NO_SPARKLE=TRUE"
 
+post-patch {
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/MacPass/NSApplication+MPAdditions.m
+}
+
 # We skip the build phase because building will also be done in the destroot phase.
 # See https://trac.macports.org/ticket/57137
-build {}
+
+# Create Resource folder beforehand, otherwise we get permission
+# errors during CompileXIB
+build {
+    xinstall -d ${destroot}/${name}.app/Contents/Resources
+}
+
+post-destroot {
+    # Copy header files needed to build plugins
+    set headerDir ${destroot}${prefix}/include/MacPass
+
+    xinstall -d ${headerDir}
+    copy ${worksrcpath}/MacPass/MPDocument.h \
+    ${worksrcpath}/MacPass/MPEntrySearchContext.h \
+    ${worksrcpath}/MacPass/MPModelChangeObserving.h \
+    ${worksrcpath}/MacPass/MPPlugin.h \
+    ${worksrcpath}/MacPass/MPTargetNodeResolving.h \
+    ${worksrcpath}/MacPass/NSString+MPPasswordCreation.h \
+    ${headerDir}
```

Comment:
You're changing the port so that it installs additional files. Therefore its revision must be increased.

Reply:
Ok. Understood and done.